translate/system-test/quickstart.test.js

```diff
@@ -18,21 +20,27 @@ const translate = proxyquire(`@google-cloud/translate`, {})({
   key: process.env.TRANSLATE_API_KEY
 });
 const string = `Hello, world!`;
+const translated = `Привет мир!`;
```

Comment:
Nit: rename to `expectedTranslation` or something like that? (i.e. denote that this is the *expected* result somehow.)